app/lib/clients/vault/identity_alias.rb

```diff
@@ -0,0 +1,68 @@
+module Clients
+  class Vault
+    module IdentityAlias
+      def put_entity_alias(entity_name, alias_name, auth_method)
+        write_identity_alias("entity", entity_name, alias_name, auth_method)
+      end
+
+      def put_group_alias(group_name, auth_method)
+        write_identity_alias("group", group_name, group_name, auth_method)
+      end
+
+      def read_group_alias(group_name, alias_name)
+        id = read_entity_alias_id(group_name, alias_name)
+        client.logical.read("identity/entity-alias/id/#{id}")
```

Comment:
group-alias?

Reply:
i think this diff isn't right? don't see in the source